### ISSUE TITLE
Missing instructions for the provided binary archives

### ISSUE THREAD

**Issue by TheDet** [WebAssembly/wabt#1420]
On one hand there is the Readme.md providing information for cloning and building the suite manually.
On the other hand there are binaries packed in archives for different platforms provided on the 'releases' tab, which is very helpful for beginners.

What I am missing though, are instructions for what to do after unpacking the binary archives.

E.g. the wabt-1.0.15-linux.tar.gz :  Extracting it gives a bunch of bin files ...  is there a special target folder to copy them into?
What about the .asm files are they meant to be copied somewhere too?

(Note: There is for example the 'Programming WebAssembly with Rust' book which starts with wabt as WebAssembly introduction, but more or less "assumes" the binaries to be in /usr/local/bin).

**Comment by binji** [comment]
Good point, this should be clarified. But to answer your question, the executables don't depend on anything else, so you should be able to unpack them anywhere convenient.

The `.wasm` files probably shouldn't be included in the releases at all -- I didn't realize we were packaging those. @sbc100 it looks like the c-api exes and resources are getting included here, is there an easy way to exclude them from releases?

**Comment by sbc100** [comment]
Yeah, we shouldn't be including them.  I guess we should disable those targets in the release builders (maybe even disable all testing?  we also don't want to ship the unittest binary right?)

**Comment by binji** [comment]
We could do that, but I don't think we have a separate release builder. I guess it could be a different configuration though?
